Library/Homebrew/formula_installer.rb

```diff
@@ -1232,6 +1232,15 @@ def pour
     with_env(HOMEBREW_RELOCATE_RPATHS: "1") do
       keg.replace_placeholders_with_locations tab.changed_files, skip_linkage: skip_linkage
     end
+
+    begin
+      bottle = formula.bottle_for_tag(Utils::Bottles.tag.to_sym)
+      return if bottle.skip_prefix_relocation?
+    rescue NoMethodError
+      nil
```

Comment:
?

What's this about?

Reply:
This was headache.  It seems on Linux the test formula used in CI doesn't have a cellar defined because the CI run fails because `cellar` is undefined.  This doesn't affect any "real" bottles because they always have this defined.  If there's a better way to deal with this I'm happy to try it!

Reply:
Yeh, I'd fix up the test bottles.

Reply:
I've gained a better understanding now of why this was failing before and it turns out it's not because of the test formula.  The real problem is that when you use `brew install bottle.tar.gz`, `formula.bottle_for_tag` seems to be undefined, I guess because we are not installing from a formula.  I ran into this problem again while trying to install a locally-built bottle and that's what made me realize what the real problem was.  

I've pushed a version now that works properly with formula installs and local bottle installs, and passes CI.  This did prompt one question - the way it's written now, we assume that the bottle was always built with the default prefix for a given tag.  Is this a reasonable assumption?  It will always be true for bottles built in homebrew-core, and I imagine it is as well for most bottles in custom taps.  I feel like a bottle built with a non-default prefix would break a lot of other logic but I'm not sure.